faithcomesbyhearing/dbp
# FCBH-655 Backend: Prettier/Eslint setup

# Description
- Installed `prettier` and `prettier-php`
- Added custom prettier fixer to `php-cs-fixer`
- Fixed all lint errors
- Added `npm run lint` and `npm run lint-fix` to fix and check lint errors
- Updated `.php_cs.dist`
- Added husky pre-commit validation in order to prevent commits with lint errors

## Issue Link
Original Story: [FCBH-655](https://fullstacklabs.atlassian.net/browse/FCBH-655) 
Review Story: [FCBH-819](https://fullstacklabs.atlassian.net/browse/FCBH-819) 

## How Do I QA This
- Run `npm install`
- Run `npm run lint` and verify that the lint validation happens
- Run `npm run lint-fix` and verify that lint errors are fixed

## Checklist
